### ISSUE TITLE
Latex Late Game Recipe Broken

### ISSUE THREAD

**Issue by ZeekDaGeek** [FTBTeam/FTB-Continuum#482]
* **Modpack Version**: 1.5.2

* **Issue**: Since AE latex isn't a thing anymore this needs to be updated:

* **Mod/s Affected**:

* **Known Fix**:

```
./scripts/industrialForegoing.zs:#alt latex recipe for later game
./scripts/industrialForegoing.zs:mods.thermalexpansion.Crucible.addRecipe(<liquid:latex> * 250, <appliedenergistics2:material:5>, 1000);
```

to

```
./scripts/industrialForegoing.zs:#alt latex recipe for later game
./scripts/industrialForegoing.zs:mods.thermalexpansion.Crucible.addRecipe(<liquid:latex> * 250, <ore:itemSilicon>, 1000);
```

**Comment by Rhajos** [comment]
I have tried this fix by my self but it tells me ingame: "a method available but none parameters". What should i do?

**Comment by tfox83** [comment]
Fixed in 1.6.0 beta
